client/src/actions/testData.json

```diff
@@ -1,225 +1,199 @@
 [
```

Comment:
Looks like this file is overriding the new testData.json file?

Its removing some items required for the company list table.

Reply:
Has the new testData.json file been merged to main? My local main is up to date and I just pulled main again into this branch to make sure.